### ISSUE TITLE
Test for WCAG failure F17: insufficient information in DOM to determine one-to-one relationships

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#162]
The objective of this technique is to ensure that Web pages can be interpreted consistently by user agents, including assistive technology. If it is ambiguous, different user agents including assistive technologies could present different information to their users. Users of assistive technology for example may have different information presented to them than users of other mainstream user agents. Some elements and attributes in markup languages are required to have unique values, and if this requirement is not honored, the result can be irregular or not uniquely resolvable content. For example, when id attribute values are not unique, they are particularly problematic when referenced by labels, headers in data tables, or used as fragment identifiers, as user agents do not have enough information to determine essential relationships (i.e., to determine which label goes with which item). 
## Test procedure
1. Check for id and accesskey values which are not unique within the document.
2. Check that attribute values that have an idref value have a corresponding id value.
3. For tables that use the axis attribute, check that all values listed in the axis attribute have a corresponding id value in a table header cell in the same table.
4. For client-side image maps, check that the value of the usemap attribute has a corresponding id value if the usemap attribute is not a URI.
## Expected results

If step 1, step 3 or step 4 is true or step 2 is false, then this failure condition applies
